desktop/cmp/panel/impl/dragger/DraggerModel.ts

```diff
@@ -30,25 +30,34 @@ export class DraggerModel extends HoistModel {
     override onLinked() {
         this.throttledSetSize = throttle(size => (this.panelModel.size = size), 50);
 
-        // Add listeners to el to ensure we can get non-passive handlers than can preventDefault()
+        // maintain listeners on el to ensure we can get non-passive handlers than can preventDefault()
         // React synthetic touch events on certain browsers (e.g. airwatch) don't yield that
         this.addReaction({
             track: () => this.ref.current,
-            run: current => {
-                if (current) this.addListeners(current);
+            run: (current, old) => {
+                this.toggleListeners(old, false);
+                this.toggleListeners(current, true);
             }
         });
     }
 
-    private addListeners(el) {
+    private toggleListeners(el, add: boolean) {
+        if (!el) return;
+        const fn = (element, event, handler, options = undefined) => {
+            add
+                ? element.addEventListener(event, handler, options)
+                : element.removeEventListener(event, handler, options);
+        };
         if (XH.isDesktop) {
-            el.addEventListener('dragstart', this.onDragStart);
-            el.addEventListener('drag', this.onDrag);
-            el.addEventListener('dragend', this.onDragEnd);
+            fn(el, 'dragstart', this.onDragStart);
+            fn(el, 'dragend', this.onDragEnd);
+            this.isFirefox // drag co-ordinates not impl in firefox -- use 'dragover' on doc
+                ? fn(document, `dragover`, this.onDrag)
```

Comment:
Cool fix, thank you for finding!

I was wondering how adding the event on document managed to not somehow trigger across different models, but I see that each model is checking if it has `resizeState` which is only set `onDragStart`.  It was a bit subtle to me dropping into this code, but seems to work beautifully.